pkg/bindings/images/types.go

```diff
@@ -0,0 +1,9 @@
+package images
+
+//go:generate go run ../generator/generator.go ImageRemove
+// ImageRemove are optional options for image removal
+type ImageRemove struct {
+	// Forces removes all containers based on the image
+	Force  *bool
```

Comment:
If we need to override a field name vs. parameter, we could put a something like
```
Force *bool `param:"FoRcE"`
```
and then write that value into the generated code for `ToParam()` use.

Reply:
will do in later pr